scripts/sanitycheck

```diff
@@ -1041,10 +1040,13 @@ class MakeGenerator:
             by execute() will be keyed by its .name field.
         """
         args = ti.test.extra_args[:]
-        if len(ti.test.extra_configs) > 0:
+        if len(ti.test.extra_configs) > 0 or options.coverage:
             args.append("OVERLAY_CONFIG=%s" %
                         os.path.join(ti.outdir, "overlay.conf"))
 
+        if ti.test.type == "unit" and options.coverage:
+            args.append("COVERAGE=1")
```

Comment:
The problem is that this path is not executed with --enable-coverage (but it is with -C)
Meaning it should be `options.enable_coverage` instead